packages/panels/resources/views/components/resources/relation-managers.blade.php

```diff
@@ -26,7 +28,10 @@
                 $tabs = $managers;
 
                 if ($content) {
-                    $tabs = array_replace([null => null], $tabs);
+                    match ($contentTabPosition) {
+                        \Filament\Support\Enums\ContentPosition::After => $tabs = $tabs + ["" => null],
```

Comment:
Can this be done with `array_merge()`, is the result the same? That would be consistent. Also with a `null` array key instead of an empty string?